server/chat/backend/agent/tools/gitlab_tool.py

```
@@ -0,0 +1,606 @@
+    mr_url = mr_resp.get("web_url", "")
+    mr_iid = mr_resp.get("iid", 0)
+
+    try:
+        with db_pool.get_admin_connection() as conn:
+            cursor = conn.cursor()
+            cursor.execute(
+                "UPDATE incident_suggestions SET pr_url=%s, pr_number=%s, created_branch=%s, applied_at=%s WHERE id=%s",
+                (mr_url, mr_iid, branch_name, datetime.now(timezone.utc), suggestion_id),
+            )
+            conn.commit()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)